xmake/rules/c++/modules/modules_support/dependency_scanner.lua

```diff
@@ -200,7 +200,9 @@ function _get_package_modules(target, package, opt)
     for _, metafile in ipairs(metafiles) do
         package_modules = package_modules or {}
         local modulefile, name, metadata = _parse_meta_info(target, metafile)
-        package_modules[name] = {file = path.join(modulesdir, modulefile), metadata = metadata}
+        local package_data = package["_INFO"]
```

Comment:
please do not access private vars. we should call `package:libraryfiles()` or `package:get()` to get them.